### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by ryyppy**

### CODE REVIEW

File: src/layouts/LandingPageLayout.res [hunk 1]
```diff
@@ -1,21 +1,356 @@
-module Link = Next.Link
```
Comment (kanishka-work) [inline src/layouts/LandingPageLayout.res:1]: From a style perspective, I haven't settled on whether I like to implement more within a layout or within a page. In this case, I would tend towards a page.

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+author: rescript-team
+date: "2021-06-25"
+title: Roadmap & New Landing Page
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:4]: it reads like
```
Roamdap & New
Landing Page
```
asymetric?

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  Announcing our new roadmap for 2021 / 2022, release cycle plans and landing page.
+---
+
+import Image from "src/components/Image.mjs"
+
+## Team Update
+
+It has almost been a year since we originally [launched our new ReScript brand](/blog/bucklescript-is-rebranding), and we think it has been a pretty successful journey so far:
+
+- An all new documentation experience
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:15]:
```suggestion
- Brand new documentation experience
```

Let's get rid of "An"

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- Relaunch of `rescript-react` and its new docs
+- Release of our re-imagined VSCode & Vim plugins
+- New syntax, compiler and build system improvements, including incremental build performance at scale
+- Improved polymorphic variant interop for numbers and strings
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:19]:
```suggestion
- Improved interop with polymorphic variants for numbers and strings
```
the word "improved" was used twice, also could be a bit more lightweight.

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- A greatly simplified object system (no need for `Js.t`)
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:20]:
```suggestion
- Accessible object system (no need for `Js.t`)
```

I don't like these sentences starting with "A". (Very subjective feedback haha)

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+- Making every part of ReScript fully Open Source and therefore independent from any company politics
+- The release of our new `rescript` npm package and cli to replace `bs-platform`
+- etc. 
+
+Of course there are many more things to come in the near future, and we wanted to make sure to keep track of our main goals in a more reliable and transparent way.
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:25]:
```suggestion
Stay tuned, this is just the beginning!
```

The second part of the sentence is a bit confusing, so I left it out.

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+### New Roadmap and Release Cycle Plans
+
+We had some thorough discussions about the future of the project and outlined the most important milestones for the next upcoming releases.
+
+**We also settled on a release cycle plan to give a better overview on estimated releases:**
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:31]: This sentence also starts with "We", just as the previous one.

```suggestion
**Here's the gist:**
```

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+- Two release channels: `stable` and `experimental`
+- More predictable release dates and better migration steps
+- Better communication and discussion for breaking changes
+- Helps the community / companies to align on major stable versions
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:35]:
```suggestion
- Help the community / companies align on major stable versions
```

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+The detailed roadmap with all our planned changes (and definition of our release cycle plan) can be found in our [community / roadmap](/community/roadmap) section.
+
+### New Landing Page
+
+After several iterations, and lots of desperate days of designing and copywriting, we are happy to announce that we are finally releasing our new landing page.
```
Comment (IwanKaramazow) [inline _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:41]:
```suggestion
After several iterations, we are happy to announce our new [landing page](https://www.rescript-lang.org).
```

File: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx (hunk 2 above)
Comment (ryyppy) [reply _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx:4]: should we go with `Roadmap 2021 & New Landing Page`?

Comment (ryyppy) [reply _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx]: should be more symmetric now

**Pull request merged**